base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -95,6 +101,8 @@ impl<B: BlockchainBackend> Clone for Validators<B> {
             block: Arc::clone(&self.block),
             orphan: Arc::clone(&self.orphan),
             horizon_state_header: Arc::clone(&self.horizon_state_header),
+            chain_gb: Arc::clone(&self.chain_gb),
```

Comment:
I've been scratching my head trying to guess what `gb` stands for. 

Is it Genesis Block?

Reply:
Yes, it is the Genesis Block. The name has been changed to `genesis_block` and `GenesisBlockValidator`.